docs/getting-started/install-agent-on-windows.md

```diff
@@ -7,6 +7,8 @@ weight = 120
 
 The installer will install Grafana Agent into the default directory `C:\Program Files\Grafana Agent`. The [windows_exporter integration](https://github.com/prometheus-community/windows_exporter) can be enabled with all default windows_exporter options.
 
+The installer can be retrieved for the latest [Grafana Agent release](https://github.com/grafana/agent/releases/latest) by downloading the release asset named `grafana-agent-installer.exe`.
```

Comment:
`The installer can be retrieved for the latest [Grafana Agent release](https://github.com/grafana/agent/releases/latest) by downloading [grafana-agent-installer.exe](https://github.com/grafana/agent/releases/latest/download/grafana-agent-installer.exe).`

Directly link?

Reply:
Hmm, I don't know, it feels like we're asking people to trust random links that download executables.

Reply:
We already do this on the Releases page for better or worse.

Reply:
WDYT about doing both to help people who are hesitant?